### PR TITLE
[WFCORE-5842] [primary/secondary] Bump the xsd to 20.0 due to documen…

### PR DESCRIPTION
**Pull request by moulalis**
Issue: https://issues.redhat.com/browse/WFCORE-5842

### CODE REVIEW

File: server/src/main/resources/schema/wildfly-config_20_0.xsd [hunk 1]
```diff
@@ -0,0 +1,3892 @@
...
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns="urn:jboss:domain:19.0"
```
Comment (moulalis) [inline server/src/main/resources/schema/wildfly-config_20_0.xsd:22]: @yersan this also be changed to 20.0 in this scope?

**Comment by jmesnil** [comment]
@moulalis could you specify what are the documentation changes? Could you also paste or attached a diff between the XSD files to help figuring out what changed? thanks

**Comment by darranl** [comment]
Actually if we have modifications to the schema can we please split them into their own commit otherwise when using IDEs and command line tools we will not be able to detect what happened.

WildFly Core may need a schema bump for other reasons but a documentation change doesn't sound like a sufficient justification unless there is also a change in behaviour.

**Comment by yersan** [comment]
> WildFly Core may need a schema bump for other reasons but a documentation change doesn't sound like a sufficient justification unless there is also a change in behaviour.

Hi @darranl , thanks for pointing it out. The original intention of this PR was just to adapt the wrong language usage used in the XSD (e.g "fetch-from-master" admin only policy value), however, the analysis doc of the related feature revealed we will need to adapt the parsers to accept "fetch-from-master" and "fetch-from-domain-controller, so it became in a non-only documentation issue.

The changes to adapt the parsers will be tracked on a different Jira. Now I'm wondering if all should be addressed by this one (WFCORE-5842) but using different commits to separate the changes. Feel free to suggest, otherwise, we will continue handling two Jiras.

**Comment by moulalis** [comment]
> @moulalis could you specify what are the documentation changes? Could you also paste or attached a diff between the XSD files to help figuring out what changed? thanks

@jmesnil PFA screenshots of the differences. https://docs.google.com/document/d/10JWg75ASnR5wE2PMytaEp1I0PbHI_UvZfyEyxZEoPks/edit#heading=h.uffl05i85a9a is the reference document.

Screenshot1
Screenshot3
Screenshot 2
Screenshot 4
Screenshot 5
Screenshot 6
Screenshot 7
Screenshot 8

**Comment by darranl** [comment]
@moulalis please split this into two commits so we have access to the history in the future.

**Review by jmesnil — CHANGES_REQUESTED** [review]
@moulalis none of the changes you made impact the schema. There is no need to bump them if only the doc is updated.

The doc can be updated in the existing 19.0 schema and we will bump it when the structure of the schema is effectively changed.

**Comment by yersan** [comment]
As discussed internally with @moulalis , closing this for now, we will open a different one including a replacement of fetch-from-master with fetch-from-domain-controller (WFCORE-5901)

**Pull request closed without merging**